intelmq/bots/BOTS

```diff
@@ -500,6 +500,13 @@
             "module": "intelmq.bots.experts.gethostbyname.expert",
             "parameters": {}
         },
+        "IDEA": {
+            "description": "Converts events into the IDEA format.",
+            "module": "intelmq.bots.experts.idea.expert",
+            "parameters": {
+                "test_mode": false
```

Comment:
Personally I do not like this at all. With the same argument we can add such a test-flag to every bot.

Are there any other opinions?